### ISSUE TITLE
support dragging between scrolling containers

### ISSUE THREAD

**Issue by bmcmahen** [bmcmahen/react-grid-dnd#4]

**Comment by sebiskver** [comment]
would be great to implement that feature, i really needed in my project and because of that lack i can't use this library

**Comment by bmcmahen** [comment]
Yeah, I'd love it too. Not sure when I'll have time to work on it, unfortunately.

**Comment by sebiskver** [comment]
> Yeah, I'd love it too. Not sure when I'll have time to work on it, unfortunately.

actually i can help you to support this feature, on every weekend i have free time, so with your small help i can contribute neccessary things.
